### ISSUE TITLE
Write a blog post about the AWS infrastructure in the pilot hubs

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/team-compass#101]
# Summary

We have recently made a major upgrade to our [`pilot-hubs` infrastructure](https://github.com/2i2c-org/pilot-hubs), and can now much more quickly and reliably deploy AWS JupyterHubs with Dask Gateway 🎉 

This feels like a good opportunity to write a short blog post that covers a few topics:

- Major reason that we wished to support AWS (and a few example hubs using it)
- Major design decisions about our AWS infrastructure setup (e.g., `kops`, `jsonnet`, etc)
- Overview of the deployment process
- Major things we'd like to improve
- ...something else?

# Actions

- [ ] Write up an outline of the major things to discuss
- [ ] Turn this into a blog post draft
- [ ] Iteration
- [ ] Publish to the 2i2c blog

**Comment by yuvipanda** [comment]
I'm not really sure we'll have the bandwidth to do this given our other commitments.

**Comment by damianavila** [comment]
As you suggesting to close it?

**Comment by choldgraf** [comment]
I think we should think about this in the context of our broader strategy - there will always be engineering things to tackle, so the question is "when do we prioritize things like blog posts over things like engineering tasks?" I'm not sure I know the answer to the question, but I don't think that it is "never" otherwise we will never publish blog posts 😅

**Comment by sgibson91** [comment]
> when do we prioritize things like blog posts over things like engineering tasks?

As someone who finds starting/completing blog posts incredibly hard and will put them off in favour of technical puzzles, THIS! ^^^ 🤣

**Comment by damianavila** [comment]
Yep, my question above was intended to start this discussion 😈  .
I do think we need to give things like blog posts enough priority to compete with other eng tasks.
It is hugely important to showcase the stuff we are doing so others can learn about them and engage with us.
